util/maputil/maputil_test.go

```diff
@@ -251,7 +251,7 @@ func TestCloneMap(t *testing.T) {
 	t.Run("NilMap", func(t *testing.T) {
 		var testMap, copyMap map[string]string = nil, nil // copyMap is a manual copy of testMap
 		clone := Clone(testMap)
-		testMap = map[string]string{"foo": "bar"}
+		testMap = map[string]string{"foo": "bar"} //nolint: ineffassign
```

Comment:
FYI: This is deleted in the Go 1.22 PR.

Reply:
I removed the `nolint` comment in https://github.com/prebid/prebid-server/commit/6977017fecf71105977c6ca7bb75cc0fac731f54.  I haven't rebased the PR branch to avoid messing up with the review - a rebased version / diff with the current `master` would be: https://github.com/prebid/prebid-server/compare/master...dmitris:prebid-server:golangci-lint03?expand=1